.github/workflows/continuous-deploy.yml

```diff
@@ -1163,6 +1158,7 @@ jobs:
             -e CLIENT_SECRET=${{ secrets.CLIENT_SECRET }} \
             -e GOVERNANCE_CLIENT_SECRET=${{ secrets.GOVERNANCE_CLIENT_SECRET }} \
             -e CLOUDAPI_URL=${{ secrets.CLOUDAPI_URL }} \
+            -e WAYPOINT_URL=${{ secrets.WAYPOINT_URL }} \
```

Comment:
@rblaine95 I added reference to waypoint_url secret here, which doesn't yet exist - please can you add 🙏

Reply:
Done 👍